dts/arm/st/f4/stm32f405.dtsi

```diff
@@ -242,4 +242,20 @@
 		#phy-cells = <0>;
 		label = "OTGHS_FS_PHY";
 	};
+
+	fmc: memory-controller@a0000000 {
```

Comment:
STM32F405 doesn't seem to have FMC according to CubeMX. Only F427/F437/F429/F439/F446/F469/F479 have the peripheral.

Reply:
@gmarull This is my bad. I looked at CMSIS definitions and FMC is defined there in F405/F407. Haven't crossed checked on CubeMx. Looking to Ref manual, FMC support is indeed limited to F42X/F43X on the "F405/07 Line".
@mkschreder Sorry for the trouble, would you mind moving this definition to F427/F437/F446/F469 SoC dtsi files ?